File: app/src/main/java/com/nagot/lastfm/ui/albuminfo/AlbumInfoFragmentMvpPresenter.java
package com.nagot.lastfm.ui.albuminfo;

import com.nagot.lastfm.base.BaseMvpPresenter;
import com.nagot.lastfm.ui.artistinfo.ArtistInfoFragmentMvpView;

/**
 * Created by IanNagot on 16/03/2018.
 */

public interface AlbumInfoFragmentMvpPresenter<V extends ArtistInfoFragmentMvpView> extends BaseMvpPresenter<V> {

    void getArtistInfo(String artistName);
}

File: app/src/main/java/com/nagot/lastfm/ui/albuminfo/AlbumInfoFragmentMvpPresenter.kt
package com.nagot.lastfm.ui.albuminfo

import com.nagot.lastfm.base.BaseMvpPresenter

/**
 * Created by Nagot on 18/03/2018.
 */
interface AlbumInfoFragmentMvpPresenter<V: AlbumInfoFragmentMvpView>: BaseMvpPresenter<V> {

    fun getAlbumInfo(albumName: String, artistName: String)
}